site/content/en/concepts/functions/_index.md

```diff
@@ -18,7 +18,7 @@ Use cases:
 - **Configuration Transformation:** e.g. Update all `PodSecurityPolicy` configurations to improve the
   security posture.
 
-![Functions Demo]("https://storage.googleapis.com/kpt-functions/docs/run.gif")
+![Functions Demo](https://storage.googleapis.com/kpt-functions/docs/run.gif)
```

Comment:
Can you remove the site/static/gif/run.gif file and regenerate docs by running `make docs`?

Reply:
I did :) 
Version pinning with generation tools was somewhat finicky. Hugo & OmniGraffle both put their version number in a lot of files.